### PR TITLE
Add DB version

### PR DESCRIPTION
**Pull request by nakajima**
In cases where we make schema changes, we should increment this number.

Eventually we should just have a migration setup in place, but this seems like a decent option for now.

### CODE REVIEW

File: xmtp-inbox-ios/Models/DB.swift [hunk 1]
```diff
@@ -68,6 +71,6 @@ class DB {
 	}
 
 	var location: URL {
-		URL.documentsDirectory.appendingPathComponent("db\(mode == .normal ? "" : "-test").sqlite")
+		URL.documentsDirectory.appendingPathComponent("db\(mode == .normal ? "" : "-test-v\(DB.version)").sqlite")
```
Comment (elisealix22) [inline xmtp-inbox-ios/Models/DB.swift:74]: Do we want the version in the `.normal` mode as well?

**Review by elisealix22 — APPROVED** [review]
Makes sene to me! I'm not too familiar on the normal vs test version, but do we want to append it to both?

File: xmtp-inbox-ios/Models/DB.swift (hunk 1 above)
Comment (nakajima) [reply xmtp-inbox-ios/Models/DB.swift:74]: Oops, great catch.

Comment (nakajima) [reply xmtp-inbox-ios/Models/DB.swift:74]: I think we might want to include the XMTP env here too, I'm seeing some crossover between dev and prod on my device.

**Pull request merged**